dev-docs/bidders/silverpush.md

```diff
@@ -124,6 +123,44 @@ var videoAdUnits = [{
 }]
 ```
 
-## Additional Details
+### First Party Data
+Publishers should use the `ortb2` method of setting First Party Data. The following fields are supported:
+
+- ortb2.user.ext.eids
```

Comment:
Prebid doesn't consider eids aren't first party data. FPD fields are: site.content.data, site.ext.data, user.data, user.ext.data.

Reply:
Fixed it 
update Commit : 
[bb20ddb](https://github.com/prebid/prebid.github.io/pull/4748/commits/bb20ddbab18a0c81c108f805b2e7614e77cabfbf)